api/v1/collection.go

```diff
@@ -75,8 +86,21 @@ func handleGetCollections(c echo.Context) error {
 //	@Produce		json
 //	@Success		200	{object}	ResponseEnvelope{response=Collection}
 //	@Router			/collections/:collectionUUID [put]
-func handleModifyCollection(c echo.Context) error {
-	return c.JSON(http.StatusNotImplemented, map[string]string{})
+func (a *apiV1) handleModifyCollection(c echo.Context) error {
+	var collection CollectionResponse
+	err := json.NewDecoder(c.Request().Body).Decode(&collection)
+
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, CreateErrorResponseEnvelope(c, http.StatusInternalServerError, err.Error()))
+	}
+
+	err = a.db.Model(&db.Collection{TenantID: db.ID(GetTenantContext(c).TenantID), CollectionID: uuid.MustParse(c.Param("collectionUUID"))}).Updates(&collection).Error
```

Comment:
i might suggest moving the `uuid` parsing out of this line and doing it separately, then you can return a separate error of `could not parse uuid` or the like. Not sure what the behavior would be if an invalid UUID is sent here, might produce an unrecognizable error